src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery.getRandomBall().print();
```

Comment:
Create one instance of `Lottery`
```suggestion
            lottery.getRandomBall().print();
```